docs/troubleshoot/app-framework/app-troubleshoot.md

```diff
@@ -1,8 +1,194 @@
 
-Before you reach out for support:
+Most of the solutions below identify issues by referring to the Zowe [logs](app-mustgather.md). To identify and resolve issues, you should become familiar with their names and locations.
 
-1. Is there already a GitHub issue (open or closed) that covers the problem? Check [Application Framework issues](https://github.com/zowe/zlux/issues).
-2. Review the current list of [Known issues](app-known-issues.md) in documentation. Also try searching using the Zowe Docs search bar.
+The Zowe Application Framework uses GitHub issues.  Check whether is an already a GitHub issue (open or closed) that covers the problem, see [https://github.com/zowe/zlux/issues](https://github.com/zowe/zlux/issues).
+
+## Desktop apps fail to load
+
+**Symptom:**
+
+When you open apps in the desktop they display a page with the message "The plugin failed to load."
+
+**Solution:**
+
+NodeJS v8.16.1 performs auto-encoding in a way that breaks Zowe apps. See [https://github.com/ibmruntimes/node/issues/142](https://github.com/ibmruntimes/node/issues/142) for details.
+
+Use a different version of NodeJS 8, such as v8.17.0, or try NodeJS version 12 which are both available at [https://www.ibm.com/ca-en/marketplace/sdk-nodejs-compiler-zos](https://www.ibm.com/ca-en/marketplace/sdk-nodejs-compiler-zos). Download the `ibm-trial-node-v8.17.0-os390-s390x.pax.Z` file.
+
+## NODEJSAPP disables immediately
+
+**Symptom:**
+
+If you receive the message `CEE5207E The signal SIGABRT was received in stderr`, you might have reached the limit for shared message queues on your LPAR.
+
+**Solution:**
+
+When Node.js applications are terminated by a SIGKILL signal, shared message queues might not be deallocated. In the IBM "[Troubleshooting Node.js applications](https://www.ibm.com/support/knowledgecenter/en/SSGMCP_5.5.0/troubleshooting/node/node-troubleshooting.html)" documentation, see the section titled **If the NODEJSAPP disables immediately**.
+
+
+## Cannot log in to the Zowe Desktop
+
+**Symptom:**
+
+When you attempt to log in to the Zowe Desktop, you receive the following error message, displayed beneath the **Username** and **Password** fields. 
+
+```
+Authentication failed for 1 types:  Types: ["zss"]
+```
+
+**Solution:**
+
+For the Zowe Desktop to work, the node server that runs under the ZWESVSTC started task must be able to make cross memory calls to the ZWESIS01 load module running under the ZWESISTC started task. If this communication fails, you see the authentication error.  
+
+There are two known problems that cause thie error to occur.  The [Zowe architecture diagram](../../getting-started/zowe-architecture.md) shows a path from the Zowe desktop ZLUX server to the zssServer (across the default port 8542) which then connects to the `ZWESIS01` started task using cross memory.  One of those two connections (ZLUX to zssServer, or zssServer to X-MEM) is likely failing.

```

Comment:
To "... started task using cross memory", add the word "communication".  BTW, the IBM KC https://www.ibm.com/support/knowledgecenter/SSLTBW_2.1.0/com.ibm.zos.v2r1.ieaa500/ieaa50039.htm uses the term "cross memory" without the hyphen.